cluster/selfplay/Dockerfile.tpu

```diff
@@ -0,0 +1,38 @@
+FROM gcr.io/tensor-go/cc-base:0.20
+
+RUN apt update && apt-get install -y --no-install-recommends \
+        build-essential \
+        curl \
+        git \
+        libcurl3-dev \
+        libfreetype6-dev \
+        libpng12-dev \
+        libzmq3-dev \
+        pkg-config \
+        rsync \
+        software-properties-common \
+        unzip \
+        zip \
+        zlib1g-dev \
+        python3 \
+        python3-numpy \
+        python3-dev \
+        python3-pip \
+        python3-wheel \
+        python3-setuptools \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+
+ADD staging/requirements.txt /app/requirements.txt
+
+RUN pip3 install --upgrade pip
+RUN pip3 install -r /app/requirements.txt
+RUN pip3 install "tensorflow==1.9.0rc2"
```

Comment:
nope, our docs aren't yet tested on tf-gpu 1.9 with whatever version of cuda; it will probably not work.  As it is, this is just the TPU dockerfile that will use TF 1.9.